ui-tests/src/test/resources/features/integrations/twitter-to-salesforce.feature

```diff
@@ -74,9 +75,10 @@ Feature: Integration - Twitter to Salesforce
     Then wait until integration "Twitter to Salesforce E2E" gets into "Running" state
     #And verify s2i build of integration "Twitter to Salesforce E2E" was finished in duration 1 min
 
-    Then tweet a message from twitter_talky to "Twitter Listener" with text "Red Hat"
+    Then tweet a message from twitter_talky to "Twitter Listener" with text "Red Hat #e2e #syndesis4ever"
+    And sleep for "30000" ms
 
     # validate salesforce contacts
-    Then check that contact from SF with last name: "Jackson" has description "Red Hat"
+    Then check that contact from SF with last name: "Jackson" has description "Red Hat #e2e #syndesis4ever"
```

Comment:
Seems like there is an issue in this validation step. The last name of the contact created in the salesforce is not really "Jackson" but "Talky". The second issue is, that the description will also contain the "@syndesis_listen" at the end.